ui/app/adapters/sync/association.js

```diff
@@ -49,8 +50,8 @@ export default class SyncAssociationAdapter extends ApplicationAdapter {
 
   // array of association data for each destination a secret is synced to
   fetchSyncStatus({ mount, secretName }) {
-    const url = `${this.buildURL()}/${mount}/${secretName}`;
-    return this.ajax(url, 'GET').then((resp) => {
+    const url = `${this.buildURL()}/destinations`;
+    return this.ajax(url, 'GET', { data: { mount, secret_name: secretName } }).then((resp) => {
```

Comment:
Purposefully not encoding query param because browser handles already.
<img width="1209" alt="Screenshot 2023-12-20 at 11 14 41 AM" src="https://github.com/hashicorp/vault/assets/68122737/5aca7532-1b16-4175-aeb7-7c5d2ccabf40">